frontend/src/admin/usersinterface.tsx

```diff
@@ -0,0 +1,4 @@
+export interface IUser {
```

Comment:
Please don't use the `Ix` naming convention. It adds no value and is a C# convention and we are writting Typscript. An interface in Typescript is not the same as an interface in C#.

Reply:
@LindvedKrvang  All other interface declarations in the code are using this naming convention.
It's a convention i made, as it makes the code easier to read (IMHO)
If TV2 wish another convention, i recommend to do that in a refactor with that as focus.
Otherwise the consistency of the code can be quite a mess.

Reply:
@olzzon What @idax6797 is working is to do a full refactor of the frontend. I think it is fine to get rid of the `I` prefix in the frontend. In this process, we can create a task for doing the refactor for the backend as well. That should be a minor task.

Reply:
@olzzon It's still a C# convention. The convention explicits state it's an interface which makes sense when you are writing in a language where interfaces can't be instantiated but only are abstractions. The prefix serves as a reminder of that fact. 
That fact, however, is not valid in Typescript and hence the `I` prefix becomes meaningless and adds no value.

Sidenote: We should strive to follow the standards of the languages we are writing in. Typescript does allow to use double quotes for strings, but the standard is to use single quotes. But the standard in C# and Java is to use double quotes. So we are using single quotes when writing Typescript and not when writing Java. By that same logic, we should not prefix interface with `I` i Typescript.

Reply:
@LindvedKrvang AFAK this is something that can split many. 
There are both guidelines point to having I* and to NOT have it, main reason for using it is for way faster grokking in the code.
Where have you read that this convention is not legit in typescript?
If TV2 decides to use it, i will of cause always follow that, in code written for TV2.

Reply:
> Where have you read that this convention is not legit in typescript?

It's perfectly valid code that will compile and runs. What I meant by `That fact, however, is not valid in Typescript` is that, unlike C# and Java, interfaces in Typescript allows for concrete objects to be intialized directly from the interface i.e. you can have code like this:
```
interface MyInterface {
  someAttribute: string
}

const myVariable: MyInterface = {
  someAttribute: 'Hello World'
}
```

This is not possible in Java and C#. There you need to make a class that implements the interface to be able use it:
```
public interface MyInterface {
  public String someMethod;
}

public class MyClass implements MyInterface {
  public String someMethod() {
    return "Hello World";
  }
}

public void main(...) {
  MyInterface myInterface = new MyClass();
}
```

So I say again: The `I` prefix is to remind you that this is an abstraction. Since Typescript allows you to make concrete objects from it, it is not an abstraction only and therefore the `I` provides no value (it acutally indicates that it is something that it is not)

Reply:
@LindvedKrvang I understand your arguments, and know where it came from, and will respect them if that's the TV2 guideline.
My point is just that I don't agree, and I can find just as many teams that have decided to use the Ix convention in typescript, because it makes the code easier to read, and makes Autosuggestions/completion faster when coding. 😊

Reply:
To each their own :)
I would still like to point out that in this specific case, the "interface" is the contract of one our entities. In C# and Java the equivalent there is a class and not an interface. So in conjuction with the `I` being misrepresentative, it's a convention that is used for a different "type" of interfaces than what we are doing here.

Sidenote: In my opinion the intellisense/autocompletion is already so fast that adding the `I` makes a negliable differences. (But that is most likely IDE dependent)

Reply:
@olzzon I get where you are comming from. As a C# developer, i have to actively remind myself to not add 'I' to interfaces, when coding Typescript.

Reply:
@andr9528 I've actually never used C#, so it's not related to that. 
It's just that I think it's easier to read the code that way 😊

Reply:
Changed from IUser to User for now.

Reply:
A bit late to the discussion. @olzzon Just to clarify. I can't speak for the entirety of TV2, but in Udviklingsteknologi we have agreed to use the convention that @LindvedKrvang points out.